reporter-web-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "@here/oss-review-toolkit-reporter",
-    "description": "React-based App to visualize scan reports generated by OSS Review Toolkit",
+    "name": "ort-web-app-reporter",
```

Comment:
commit message: "Rename package name" -> how about just "Rename the package"

Reply:
Fixed